### ISSUE TITLE
Lobby Hosted Game Are Not De-Listed When Closed

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#6025]
**How can the problem be recreated?**

- connect to lobby
- host a game

Do any of:
1) click cancel
2) click quit
3) launch game, then quit ( 'File' > 'exit program')

- notice in all 3 cases that the lobby does not de-list the hosted game right away.

**When the problem is fixed, what should have happened instead?**
Hosted game should be removed from lobby list in timely manner.

**Which Engine Version are you using?** 
2.0 @ cd8847e (Sat Mar 7 20:00:37 2020 -0800)

**Comment by prastle** [comment]
Actually they do close/de-list it just takes a minute or so to occur for some reason.  Hope this helps.  @DanVanAtta

**Comment by RoiEXLab** [comment]
@DanVanAtta If @prastle's observation is true all the time, that means that we don't properly close the Websocket connection, and are waiting for it to timeout instead.

**Comment by DanVanAtta** [comment]
Thanks for the input, I can elaborate a bit on this issue and am working on a fix. The problem was introduced when we went from http polling to websockets communication for game listing. Before https://github.com/triplea-game/triplea/pull/6007 "Expire games" was merged just a few days ago, games were not removed at all. I think previously we had a far smaller TTL and it appears we relied on that exclusively. Now the TTL is longer so the problem is more noticeable. The issue is more we are not closing the game out properly when a websocket connection is closed, rather than not closing the connection. IE: the websocket close is not connected to closing the game, it does not trigger notifications correctly.
